hera_pspec/tests/test_pstokes.py

```diff
@@ -75,5 +75,12 @@ def test_construct_pstokes(self):
         # check except for same polarizations
         nt.assert_raises(AssertionError, pstokes.construct_pstokes, dset1=uvd1, dset2=uvd1, pstokes='pI')
 
+    def test_choose_inp_dsets(self):
```

Comment:
It'd be good to add a couple more tests to handle situations where there might be multiple ways of building the same Stokes parameter.

Reply:
sure thing